### PR TITLE
Rename GeneralisedNewtypeDeriving to GeneralizedNewtypeDeriving

### PR DESCRIPTION
**Pull request by newhoggy**
Choose standard spelling for language pragmas.

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Byron/Commands.hs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```
Comment (erikd) [inline cardano-cli/src/Cardano/CLI/Byron/Commands.hs:1]: Which standard? This just switches from British to American spelling.

Comment (newhoggy) [reply cardano-cli/src/Cardano/CLI/Byron/Commands.hs:1]: Stylish Haskell crashes on the British spelling, which was introduced in `8.2`

Comment (newhoggy) [reply cardano-cli/src/Cardano/CLI/Byron/Commands.hs:1]:
```
Language.Haskell.Stylish.Parse.parseModule: could not parse ./cardano-config/src/Cardano/TracingOrphanInstances/Shelley.hs: ParseFailed (SrcLoc "<unknown>.hs" 159 18) "GeneralizedNewtypeDeriving language extension is not enabled. Please add {-# LANGUAGE GeneralizedNewtypeDeriving #-} pragma at the top of your module."
```

Comment (newhoggy) [reply cardano-cli/src/Cardano/CLI/Byron/Commands.hs:1]: I've raised a PR against `stylish-haskell`:

https://github.com/jaspervdj/stylish-haskell/issues/295

Comment (dcoutts) [reply cardano-cli/src/Cardano/CLI/Byron/Commands.hs:1]: Clearly the British spelling is the correct one :wink:

**Pull request closed without merging**